packages/northstar-wrapper/src/index.ts

```diff
@@ -1,8 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import * as northStar from '@fluentui/react-northstar';
+import {
```

Comment:
Any concerns that we don't do import * ? is it for treeshaking

Reply:
The question is why we would do `*` at all. We don't want to allow exports from `n*` so we want to keep this to a minimal set such that we don't allow developers to accidentally introduce new reliances on northstar

Reply:
Nothing needs done it will automatically work - but actually wih it didn't. Someone _**should not**_ be adding to this list. We want to get further away from n* not be more reliant on it,

Reply:
Are we saying we developers could import NorthStar component from our package? I guess we don't expose them as a public api right?

Reply:
nope, just internal developers to this repo, not npm package users